### PR TITLE
Update Holybro

### PR DESCRIPTION
**Pull request by haslinghuis**
- Add SPA06 / SPL07 support where applicable 
- Cleanup configurations
    - Update timer map
    - Remove dashboard
    - Add missing timer map for CAMERA_CONTROL on KAKUTEF7V2
    - Remove gyro 2 SPI instance on KAKUTEF7MINIV3
    - Many additional fixes by @ot0tot

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]
- approving for workflow; requesting @ot0tot review

File: configs/KAKUTEF4/config.h [hunk 1]
```diff
@@ -74,15 +72,14 @@
...
-    TIMER_PIN_MAP( 0, PC7 , 2,  0) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PA3 , 1,  1) \
-    TIMER_PIN_MAP( 4, PA2 , 1,  0) \
-    TIMER_PIN_MAP( 5, PA0 , 2,  0) \
-    TIMER_PIN_MAP( 6, PC8 , 2,  1) \
-    TIMER_PIN_MAP( 7, PA1 , 2,  0)
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN,    2,  0) \
```
Comment (ot0tot) [inline configs/KAKUTEF4/config.h:75]:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN,    2, -1) \
```

File: configs/KAKUTEF4V2/config.h [hunk 2]
```diff
@@ -76,30 +78,27 @@
...
 
 #define TIMER_PIN_MAPPING \
-    TIMER_PIN_MAP( 0, PC7 , 2,  0) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PA3 , 1,  1) \
-    TIMER_PIN_MAP( 4, PA2 , 1,  0) \
-    TIMER_PIN_MAP( 5, PC8 , 2,  0)
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN,    2,  0) \
```
Comment (ot0tot) [inline configs/KAKUTEF4V2/config.h:81]:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN,    2, -1) \
```

File: configs/KAKUTEF7/config.h [hunk 3]
```diff
@@ -79,33 +81,29 @@
...
-    TIMER_PIN_MAP( 0, PE13, 1,  1) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PE9 , 1,  2) \
-    TIMER_PIN_MAP( 4, PE11, 1,  1) \
-    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
-    TIMER_PIN_MAP( 6, PA3 , 2,  0) \
-    TIMER_PIN_MAP( 7, PD12, 1,  0)
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN   , 1,  1) \
```
Comment (ot0tot) [inline configs/KAKUTEF7/config.h:84]:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN   , 1,  -1) \
```
RX_PPM should probably be removed as the timer conflicts with Motors 3 and 4 and would not be usable anyway.

File: configs/KAKUTEF7HDV/config.h [hunk 4]
```diff
@@ -76,33 +78,28 @@
...
-    TIMER_PIN_MAP( 0, PE13, 1,  1) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PE9 , 1,  2) \
-    TIMER_PIN_MAP( 4, PE11, 1,  1) \
-    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
-    TIMER_PIN_MAP( 6, PA3 , 2,  0) \
-    TIMER_PIN_MAP( 7, PD12, 1,  0)
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN   , 1,  1) \
```
Comment (ot0tot) [inline configs/KAKUTEF7HDV/config.h:81]:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN   , 1,  -1) \
```
Same comment on removing RX_PPM

Comment (haslinghuis) [reply configs/KAKUTEF7HDV/config.h]: Done :+1:

File: configs/KAKUTEF7MINI/config.h [hunk 5]
```diff
@@ -81,29 +82,26 @@
...
-    TIMER_PIN_MAP( 0, PE13, 1,  1) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PE9 , 1,  2) \
-    TIMER_PIN_MAP( 4, PE11, 1,  1) \
-    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
-    TIMER_PIN_MAP( 6, PA3 , 2,  0) \
-    TIMER_PIN_MAP( 7, PD12, 1,  0) \
-    TIMER_PIN_MAP( 8, PB3 , 1,  0)
+    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  1) \
```
Comment (ot0tot) [inline configs/KAKUTEF7MINI/config.h:85]:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  -1) \
```
Same comment on removing RX_PPM

Comment (haslinghuis) [reply configs/KAKUTEF7MINI/config.h]: Also applied to remaining F7 boards. :+1:

File: configs/KAKUTEF7MINIV3/config.h [hunk 6]
```diff
@@ -80,30 +81,27 @@
...
-    TIMER_PIN_MAP( 6, PB7 , 1,  0) \
-    TIMER_PIN_MAP( 7, PA8 , 1,  2) \
-    TIMER_PIN_MAP( 8, PA3 , 3, -1)
+    TIMER_PIN_MAP( 0, MOTOR1_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 1, MOTOR2_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 2, MOTOR3_PIN        , 1,  0) \
+    TIMER_PIN_MAP( 3, MOTOR4_PIN        , 1,  0) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN        , 1,  0) \
+    TIMER_PIN_MAP( 5, MOTOR6_PIN        , 1,  0) \
+    TIMER_PIN_MAP( 6, CAMERA_CONTROL_PIN, 1,  0) \
```
Comment (ot0tot) [inline configs/KAKUTEF7MINIV3/config.h:90]:
```suggestion
    TIMER_PIN_MAP( 6, CAMERA_CONTROL_PIN, 1,  -1) \
```

Comment (haslinghuis) [reply configs/KAKUTEF7MINIV3/config.h]: Done for all KAKUTE targets, :1st_place_medal:

File: configs/KAKUTEF7V2/config.h [hunk 7]
```diff
@@ -80,31 +82,29 @@
...
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  1) \
+    TIMER_PIN_MAP( 1, MOTOR1_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 2, MOTOR2_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 3, MOTOR3_PIN        , 1,  2) \
+    TIMER_PIN_MAP( 4, MOTOR4_PIN        , 1,  1) \
+    TIMER_PIN_MAP( 5, MOTOR5_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 6, MOTOR6_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 7, LED_STRIP_PIN     , 1,  0) \
+    TIMER_PIN_MAP( 8, CAMERA_CONTROL_PIN, 1,  0)
```
Comment (ot0tot) [inline configs/KAKUTEF7V2/config.h]: E13 is assigned to both RX_PPM and CAMERA_CONTROL

File: configs/KAKUTEF7V2/config.h [hunk 8]
```diff
@@ -80,31 +82,29 @@
...
-    TIMER_PIN_MAP( 0, PE13, 1,  1) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PE9 , 1,  2) \
-    TIMER_PIN_MAP( 4, PE11, 1,  1) \
-    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
-    TIMER_PIN_MAP( 6, PA3 , 2,  0) \
-    TIMER_PIN_MAP( 7, PD12, 1,  0)
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  1) \
```
Comment (ot0tot) [inline configs/KAKUTEF7V2/config.h]:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  -1) \
```
Remove RX_PPM as it conflicts with timer for Motors 3 and 4

File: configs/KAKUTEF7V2/config.h (hunk 7 above)
Comment (haslinghuis) [reply configs/KAKUTEF7V2/config.h]: Remove RX_PPM_PIN :+1:

File: configs/KAKUTEF7V2/config.h (hunk 8 above)
Comment (haslinghuis) [reply configs/KAKUTEF7V2/config.h]: Done above. Please refresh.

File: configs/KAKUTEH7/config.h [hunk 9]
```diff
@@ -84,45 +85,42 @@
...
+    TIMER_PIN_MAP( 0, MOTOR1_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 1, MOTOR2_PIN        , 2,  1) \
+    TIMER_PIN_MAP( 2, MOTOR3_PIN        , 1,  2) \
+    TIMER_PIN_MAP( 3, MOTOR4_PIN        , 1,  3) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN        , 2,  4) \
+    TIMER_PIN_MAP( 5, MOTOR5_PIN        , 2,  5) \
+    TIMER_PIN_MAP( 6, MOTOR5_PIN        , 2,  6) \
+    TIMER_PIN_MAP( 7, MOTOR8_PIN        , 2,  7) \
+    TIMER_PIN_MAP( 8, LED_STRIP_PIN     , 1, 14) \
+    TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 1, 12)
```
Comment (ot0tot) [inline configs/KAKUTEH7/config.h]:
```suggestion
    TIMER_PIN_MAP( 0, MOTOR1_PIN        , 2,  0) \
    TIMER_PIN_MAP( 1, MOTOR2_PIN        , 2,  1) \
    TIMER_PIN_MAP( 2, MOTOR3_PIN        , 1,  2) \
    TIMER_PIN_MAP( 3, MOTOR4_PIN        , 1,  3) \
    TIMER_PIN_MAP( 4, MOTOR5_PIN        , 2,  4) \
    TIMER_PIN_MAP( 5, MOTOR6_PIN        , 2,  5) \
    TIMER_PIN_MAP( 6, MOTOR7_PIN        , 2,  6) \
    TIMER_PIN_MAP( 7, MOTOR8_PIN        , 2,  7) \
    TIMER_PIN_MAP( 8, LED_STRIP_PIN     , 1, 14) \
    TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 1, -1)
```

File: configs/KAKUTEH7/config.h [hunk 10]
```diff
@@ -84,45 +85,42 @@
...
-#define TIMUP5_DMA_OPT      0
-#define TIMUP8_DMA_OPT      1
+#define SPI1_TX_DMA_OPT             13
+#define ADC1_DMA_OPT                 8
+#define ADC3_DMA_OPT                 9
+#define TIMUP1_DMA_OPT               0
+#define TIMUP2_DMA_OPT               0
+#define TIMUP3_DMA_OPT               2
+#define TIMUP4_DMA_OPT               0
+#define TIMUP5_DMA_OPT               0
```
Comment (ot0tot) [inline configs/KAKUTEH7/config.h:106]:
```suggestion
#define TIMUP5_DMA_OPT               3
```

File: configs/KAKUTEH7MINI/config.h [hunk 11]
```diff
@@ -84,44 +86,41 @@
...
+    TIMER_PIN_MAP( 0, MOTOR1_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 1, MOTOR2_PIN        , 2,  1) \
+    TIMER_PIN_MAP( 2, MOTOR3_PIN        , 1,  2) \
+    TIMER_PIN_MAP( 3, MOTOR4_PIN        , 1,  3) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN        , 2,  4) \
+    TIMER_PIN_MAP( 5, MOTOR6_PIN        , 2,  5) \
+    TIMER_PIN_MAP( 6, MOTOR7_PIN        , 2,  6) \
+    TIMER_PIN_MAP( 7, MOTOR8_PIN        , 2,  7) \
+    TIMER_PIN_MAP( 8, LED_STRIP_PIN     , 1, 14) \
+    TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 1, 12)
```
Comment (ot0tot) [inline configs/KAKUTEH7MINI/config.h]:
```suggestion
    TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 1, -1)
```

File: configs/KAKUTEH7/config.h (hunk 9 above)
Comment (haslinghuis) [reply configs/KAKUTEH7/config.h]: Good catch, copy paste error on my behalf.

File: configs/KAKUTEH7MINI/config.h [hunk 12]
```diff
@@ -84,44 +86,41 @@
...
-#define TIMUP5_DMA_OPT      0
-#define TIMUP8_DMA_OPT      1
+#define SPI1_TX_DMA_OPT              13
+#define ADC1_DMA_OPT                 8
+#define ADC3_DMA_OPT                 9
+#define TIMUP1_DMA_OPT               0
+#define TIMUP2_DMA_OPT               0
+#define TIMUP3_DMA_OPT               2
+#define TIMUP4_DMA_OPT               0
+#define TIMUP5_DMA_OPT               0
```
Comment (ot0tot) [inline configs/KAKUTEH7MINI/config.h:107]:
```suggestion
#define TIMUP5_DMA_OPT               3
```

File: configs/KAKUTEH7V2/config.h [hunk 13]
```diff
@@ -80,44 +82,40 @@
...
+    TIMER_PIN_MAP( 0, MOTOR1_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 1, MOTOR2_PIN        , 2,  1) \
+    TIMER_PIN_MAP( 2, MOTOR3_PIN        , 1,  2) \
+    TIMER_PIN_MAP( 3, MOTOR4_PIN        , 1,  3) \
+    TIMER_PIN_MAP( 4, MOTOR5_PIN        , 2,  4) \
+    TIMER_PIN_MAP( 5, MOTOR6_PIN        , 2,  5) \
+    TIMER_PIN_MAP( 6, MOTOR7_PIN        , 2,  6) \
+    TIMER_PIN_MAP( 7, MOTOR8_PIN        , 2,  7) \
+    TIMER_PIN_MAP( 8, LED_STRIP_PIN     , 1, 14) \
+    TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 1, 12)
```
Comment (ot0tot) [inline configs/KAKUTEH7V2/config.h]:
```suggestion
    TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 1, -1)
```

File: configs/KAKUTEH7V2/config.h [hunk 14]
```diff
@@ -80,44 +82,40 @@
...
-#define TIMUP5_DMA_OPT      0
-#define TIMUP8_DMA_OPT      1
+#define SPI1_TX_DMA_OPT              13
+#define ADC1_DMA_OPT                 8
+#define ADC3_DMA_OPT                 9
+#define TIMUP1_DMA_OPT               0
+#define TIMUP2_DMA_OPT               0
+#define TIMUP3_DMA_OPT               2
+#define TIMUP4_DMA_OPT               0
+#define TIMUP5_DMA_OPT               0
```
Comment (ot0tot) [inline configs/KAKUTEH7V2/config.h:103]:
```suggestion
#define TIMUP5_DMA_OPT               3
```

File: configs/KAKUTEH7V2/config.h (hunk 13 above)
Comment (haslinghuis) [reply configs/KAKUTEH7V2/config.h]: This was already fixed :)

**Review by ot0tot — APPROVED** [review]

**Pull request merged**